pkg/cloud-provider/cloudapi/aws/aws_cloudinterface_impl.go

```diff
@@ -97,8 +97,8 @@ func (c *awsCloud) AddAccountResourceSelector(accNamespacedName *types.Namespace
 }
 
 // RemoveAccountResourcesSelector removes account specific resource selector.
-func (c *awsCloud) RemoveAccountResourcesSelector(accNamespacedName *types.NamespacedName, selectorName string) {
-	c.cloudCommon.RemoveSelector(accNamespacedName, selectorName)
+func (c *awsCloud) RemoveAccountResourcesSelector(accNamespacedName *types.NamespacedName, selectorNamespacedName string) {
```

Comment:
Do we need it as selectorNamespacedName to be passed as string?
Can we change it to NamespacedName type?

Reply:
Using NamespacedName type